react/components/loaders/FetchMoreButton.js

```diff
@@ -27,7 +27,7 @@ const useShowButton = (to, products, loading, recordsFiltered) => {
 }
 
 function shouldNotIncludeMap(map) {
-  if (!map || map === 'b' || map === 'brand') {
+  if (!map || map === 'b' || map === 'brand' || map === 'department') {
```

Comment:
I think you should include `map === 'c'` as well, since in most cases the map for department filter is `c`. `category-1` is also a possible option

Reply:
Added!